.eslintrc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
 		es2020: true,
 		node: true,
 	},
-	extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
+	extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended',"prettier"],
```

Comment:
Since we're using eslint --fix for formatting, do we need prettier?

Reply:
Removed